framework/src/main/java/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -1452,6 +1451,14 @@ private boolean handledByValueChecker(AnnotatedTypeMirror type) {
             */
             return COVERED_CLASS_STRINGS.contains(tm.toString());
         }
+
+        @Override
+        public Void visitConditionalExpression(
+                ConditionalExpressionTree node, AnnotatedTypeMirror annotatedTypeMirror) {
+            // Work around for https://github.com/typetools/checker-framework/issues/602.
+            annotatedTypeMirror.replaceAnnotation(UNKNOWNVAL);
+            return super.visitConditionalExpression(node, annotatedTypeMirror);
```

Comment:
TreeAnnotators don't visit their subtrees, so calling super does nothing but return null.  This method should not call super like the others in this class.  If you leave the super call, a reader has to figure out that it does nothing.